lib/tasks/import.rake

```diff
@@ -1,24 +1,20 @@
 # frozen_string_literal: true
 
-require "tasks/whitehall_news_importer"
+require "tasks/whitehall_importer"
 
 namespace :import do
   desc "Import news documents from JSON e.g. import:whitehall_news INPUT=export_file"
```

Comment:
Just a tiny one, I guess we'll need to remove the references to news from this description 😸

Reply:
Ignore - I see this is done in a later commit!

Reply:
Good spot.  It's in the wrong commit.  I'll move that change.